src/sbt-test/sbt-conductr/sandbox-conductr-roles/build.sbt

```diff
@@ -21,25 +21,24 @@ BundleKeys.roles := Set("bundle-role-1", "bundle-role-2")
 
 val checkConductrRolesByBundle = taskKey[Unit]("Check that the bundle roles are used if no sandbox roles are specified.")
 checkConductrRolesByBundle := {
-  for (i <- 0 to 2) {
-    val content = s"docker inspect --format='{{.Config.Env}}' cond-$i".!!
-    val expectedContent = "CONDUCTR_ROLES=bundle-role-1,bundle-role-2"
-    content should not include(expectedContent)
+  val psOutput = s"ps ax".lines_!.toList
```

Comment:
Would it make sense to add bundle roles to `sandbox ps` and using it instead of `ps aux`, thus making this test a bit more portable?